frontend/src/views/admin/matching/applicant-view.tsx

```diff
@@ -0,0 +1,666 @@
+import React from "react";
+import classNames from "classnames";
+
+import { Position, Application } from "../../../api/defs/types";
+import { ApplicantSummary, Match } from "./types";
+
+import { FaFilter, FaTable, FaTh, FaLock } from "react-icons/fa";
+import { BsInfoCircleFill, BsStarFill } from "react-icons/bs";
+import { RiStickyNoteFill } from "react-icons/ri";
+
+import ToggleButton from "react-bootstrap/ToggleButton";
+import ToggleButtonGroup from "react-bootstrap/ToggleButtonGroup";
+import Collapse from 'react-bootstrap/Collapse';
+import { Form, Table, Dropdown, Modal, Button } from "react-bootstrap";
+
+import { sum } from "../../../api/mockAPI/utils";
```

Comment:
We shouldn't be loading anything from the mockAPI in production code. If you need this function, move it to, e.g.,  `libs/utils`